public/app/features/explore/Explore.test.tsx

```diff
@@ -225,4 +233,14 @@ describe('Explore', () => {
       expect(dataSourcePicker).toBeInTheDocument();
     });
   });
+
+  describe('Content Outline', () => {
+    it('should retrieve the last visible state from local storage', async () => {
+      const getBoolMock = jest.spyOn(store, 'getBool').mockReturnValue(false);
```

Comment:
Not sure if this is the best option in terms of clarity. Yes, we invoke getBool in content outline and getBool is just a a wrapper around window.local.storage, but maybe we can spy on the local storage directly and look up the key. Or maybe do both?

Reply:
I think it's also fine this way. Sometimes we do it directly with local storage which is also fine, but you also need to remember to reset local storage state after the test.